src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");

```

Comment:
The message 'Truck started working' is not very informative. It should indicate what type of Truck is working. Consider including specific details about the Truck, such as its model or capacity, to make the log message more informative.